marqeta.gemspec

```diff
@@ -31,6 +31,7 @@ Gem::Specification.new do |spec|
 
   spec.add_development_dependency "bundler", "~> 1.16"
   spec.add_development_dependency "rake", "~> 10.0"
-  spec.add_development_dependency "minitest", "~> 5.0"
+  spec.add_development_dependency "rspec", "~> 3.5"
+  spec.add_development_dependency "rest-client", "~> 1.8"
```

Comment:
isn't this an 'always' dependency, not just development?

Reply:
duh 🤦‍♂️

Reply:
@andyw8 fixed